examples/getting_started/quickstart.ipynb

```diff
@@ -0,0 +1,535 @@
+     ]
+    }
+   ],
+   "source": [
+    "from pydantic import BaseModel\n",
+    "\n",
+    "\n",
+    "class Capital(BaseModel):\n",
+    "    city: str\n",
+    "    country: str\n",
+    "\n",
+    "\n",
+    "@openai.call(\"gpt-4o-mini\", response_model=Capital)\n",
+    "@prompt_template(\"Provide the capital of {country}\")\n",
+    "def get_capital_info(country: str): ...\n",
+    "\n",
+    "\n",
+    "result = get_capital_info(\"Japan\")\n",
+    "print(f\"The capital of {result.country} is {result.city}\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "For more details on response models, including advanced validation techniques, check our [Learn documentation on Response Models](https://docs.mirascope.io/learn/response-models)."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Output Parsers\n",
+    "\n",
+    "Output parsers allow you to process LLM responses in custom formats."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 17,
+   "metadata": {
+    "ExecuteTime": {
+     "end_time": "2024-09-06T14:00:28.482626Z",
+     "start_time": "2024-09-06T14:00:27.777878Z"
+    }
+   },
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "country_info=CountryInfo(country='Australia', capital='Canberra')\n",
+      "The capital of Australia is Canberra\n"
+     ]
+    }
+   ],
+   "source": [
+    "class CountryInfo(BaseModel):\n",
+    "    country: str\n",
+    "    capital: str\n",
+    "\n",
+    "\n",
+    "def parse_country_and_capital(response: openai.OpenAICallResponse) -> CountryInfo:\n",
+    "    country, capital = response.content.split(\" -> \")\n",
+    "    return CountryInfo(country=country, capital=capital)\n",
+    "\n",
+    "\n",
+    "@openai.call(\"gpt-4o-mini\", output_parser=parse_country_and_capital)\n",
+    "@prompt_template(\n",
+    "    \"What is the capital of {country}? Your output MUST follow the format 'country -> capital'.\"\n",
+    ")\n",
+    "def country_and_capital(country: str): ...\n",
+    "\n",
+    "\n",
+    "country_info = country_and_capital(\"Australia\")\n",
+    "print(f\"{country_info=}\")\n",
+    "print(f\"The capital of {country_info.country} is {country_info.capital}\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "For more information on output parsers, see our [Learn documentation on Output Parsers](https://docs.mirascope.io/learn/output-parsers)."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "This concludes our Quickstart Guide to Mirascope. We've covered the main features of the library, including prompt templates, basic calls, streaming, asynchronous processing, JSON mode, response models, and output parsers. Each of these features can be combined and customized to create powerful, flexible AI applications.\n",
```

Comment:
nit: make sure to update the order of topics covered here to match the updated order